README.md

```diff
@@ -1,3 +1,7 @@
+## Important note - do not use Pip version 10.0!
+
+Pip version 10 is incompatible with current versions of `mbed cli`.
```

Comment:
I would suggest adding _why_ upgrading to pip v10 is not supported. 

Mainly, this PR: https://github.com/pypa/pip/pull/4700